app/controllers/user_programme_enrolments_controller.rb

```diff
@@ -14,6 +14,18 @@ def create
     end
   end
 
+  def update
+    user_programme_enrolment = current_user.user_programme_enrolments.find(params[:id])
+
+    if user_programme_enrolment.update(user_prgoramme_enrolment_update_params)
```

Comment:
It's a good job this method name was typo'd in both places so it still worked. Let's maybe spell programme (not prgoramme) correctly though 😂